{"name": "test_send_news_success[case0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'hello pythonhello python' == 'hello python'\n  - hello python\n  + hello pythonhello python", "trace": "self = <testcase.test_05personal.TestPersonal object at 0x11bb30350>\ncase = {'message': 'hello python', 'title': '是好友，私信发送成功'}\npersonal_setup = (<page.homework_page.HomeworkPage object at 0x11bb2c590>, <page.personal_page.PersonalPage object at 0x11b8d9fd0>)\n\n    @pytest.mark.parametrize('case',personal_data.personal_data_send_success)\n    def test_send_news_success(self,case,personal_setup):\n        homework_page = personal_setup[0]\n        personal_page = personal_setup[1]\n        # 1.选择好友\n        personal_page.choose_pass_people()\n        time.sleep(2)\n        # 2,定位输入框，点击发送\n        personal_page.send_news(case[\"message\"])\n        # 3.获取页面的上发送内容\n        res = personal_page.get_send_content()\n        # 4.断言\n        try:\n            assert res == case[\"message\"]\n        except AssertionError as e:\n            log.error(\"用例--{}--执行失败\".format(case['title']))\n            log.exception(e)\n>           raise e\n\ntestcase/test_05personal.py:124: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <testcase.test_05personal.TestPersonal object at 0x11bb30350>\ncase = {'message': 'hello python', 'title': '是好友，私信发送成功'}\npersonal_setup = (<page.homework_page.HomeworkPage object at 0x11bb2c590>, <page.personal_page.PersonalPage object at 0x11b8d9fd0>)\n\n    @pytest.mark.parametrize('case',personal_data.personal_data_send_success)\n    def test_send_news_success(self,case,personal_setup):\n        homework_page = personal_setup[0]\n        personal_page = personal_setup[1]\n        # 1.选择好友\n        personal_page.choose_pass_people()\n        time.sleep(2)\n        # 2,定位输入框，点击发送\n        personal_page.send_news(case[\"message\"])\n        # 3.获取页面的上发送内容\n        res = personal_page.get_send_content()\n        # 4.断言\n        try:\n>           assert res == case[\"message\"]\nE           AssertionError: assert 'hello pythonhello python' == 'hello python'\nE             - hello python\nE             + hello pythonhello python\n\ntestcase/test_05personal.py:120: AssertionError"}, "attachments": [{"name": "log", "source": "04f5c28d-368d-4bf0-a272-81dfb8a91966-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "985ac1b2-7604-4a11-b28d-5b74eea20671-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'title': '是好友，私信发送成功', 'message': 'hello python'}"}], "start": 1604462204815, "stop": 1604462207078, "uuid": "a737a94e-1672-4d1c-860e-3b4a9d848263", "historyId": "e824cdc39e651bc199a94f51401990f1", "testCaseId": "5b723fe91b0d95a34587e4aef487bccc", "fullName": "testcase.test_05personal.TestPersonal#test_send_news_success", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_05personal"}, {"name": "subSuite", "value": "TestPersonal"}, {"name": "host", "value": "ZBMAC-C02WD0XQH.local"}, {"name": "thread", "value": "55631-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_05personal"}]}